lib/presentation/screens/albums_screen.dart

```diff
@@ -0,0 +1,226 @@
+                      child: Text(
+                        'Text',
+                        style: TextStyle(
+                          // change to best version of text style in main !!!
+                          fontSize: 20,
+                          fontWeight: FontWeight.bold,
+                          color: Colors.black,
+                        ),
+                      ),
+                    ),
+                  ],
+                );
+              },
+            ),
+          ), // for Favorites widget
+
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              TextButton(
+                style: TextButton.styleFrom(
+                  textStyle: const TextStyle(fontSize: 20),
+                ),
+                onPressed: () {},
+                child: Text(
+                  'Recent',
+                  style: TextStyle(
+                    // change to best version of text style in main !!!
+                    fontSize: 20,
+                    fontWeight: FontWeight.bold,
+                    color: Colors.black,
+                  ),
+                ),
+              ), // for Recent widget
+              TextButton(
+                style: TextButton.styleFrom(
+                  textStyle: const TextStyle(fontSize: 20),
+                ),
+                onPressed: () {},
+                child: Text(
+                  'All',
+                  style: TextStyle(
+                    // change to best version of text style in main !!!
+                    fontSize: 20,
+                    fontWeight: FontWeight.bold,
+                    color: Colors.black,
+                  ),
+                ),
+              ), // for Recent widget
+            ],
+          ),
+          SizedBox(
+            height: 210,
+            child: ListView.separated(
+              scrollDirection: Axis.horizontal,
+              padding: const EdgeInsets.all(12),
+              itemCount: 5,
+              separatorBuilder: (context, index) {
+                return const SizedBox(width: 12);
+              },
+              itemBuilder: (context, index) {
+                return Column(
+                  children: [
+                    PhotoView(width: _width, height: _heigth),
+                    Padding(
+                      padding: const EdgeInsets.all(5),
+                      child: Text(
+                        'Text',
+                        style: TextStyle(
+                          // change to best version of text style in main !!!
+                          fontSize: 20,
+                          fontWeight: FontWeight.bold,
+                          color: Colors.black,
+                        ),
+                      ),
+                    ),
+                  ],
+                );
+              },
+            ),
+          ), // for Recent widget
+        ],
+      ),
+    );
+  }
+}
```

Comment:
very good use of commas in this file. Take it as and example, but dont forget about flutter format command